source/sass/component/_imageinput.scss

```diff
@@ -0,0 +1,42 @@
+.c-imageinput {
+  .c-imageinput__preview {
+    display: block;
+
+    .c-imageinput__image {
```

Comment:
.c-imageinput__preview and .c-imageinput__image should be seperated (no need to nest them) unless there is a reason why image is dependent on a preview parent